src/framebuffer.rs

```diff
@@ -199,6 +199,12 @@ impl Device {
             .BindFramebuffer(__gl::DRAW_FRAMEBUFFER, framebuffer.0);
     }
 
+    /// Bind a framebuffer for read commands.
+    pub unsafe fn bind_read_framebuffer(&self, framebuffer: Framebuffer) {
```

Comment:
Related to the other conversation, I consciously opted to require the user to bind the read framebuffer before using `copy_attachment_to_` methods, rather than passing the framebuffer as input. This is more consistent with how the draw framebuffer is treated.

Reply:
Agree, I prefer the explicit binding as well